proposal/Eric860730/README.rst

```diff
@@ -0,0 +1,180 @@
+
+Problem to Solve
+----------------------------
+| In recent years, COVID-19 has been raging around the world.
+| It has a great impact on everyone's lives.
+| According to this article[1], the spread of COVID-19 is increasing exponentially.
+| In order to prevent the spread of COVID-19, each country/region has implemented many policies (e.g. quarantine, restriction of in-store dining, lockdown...).
+| These policies have produced different results in controlling the spread of COVID-19.
+
+
+| I want to simulate the impact of different policies on the spread of pandemic viruses.
+ User can adjust parameters(e.g. virus infection rate, virus mortality rate...) and policy to observe the final result.
+ I will use animation to show the process of infection and print the distribution of the virus spread.
+| Therefore, we can observe the difference in the distribution of infection under different policies.
+
+Prospective Users
+----------------------------
+| Those who want to know if these policies help prevent the spread of COVID-19.
+| Users can set different conditions, such as the total number of people, virus infection rate, death rate, and different policies.
+| Then observe the distribution of the virus spread and the number of deaths.
+
+System Architecture
+----------------------------
+
+Input
+^^^^^^^^^^^^
+* total number of people (option, default = 1000(<= 2000))
+* init number of infected people (**percentage** or **int**) (option, default = 1%)
+* moving speed of people (option, default = 1)
+* virus infection rate (option, default = 70%)
+* virus mortality rate (option, default = 30%)
+* recovery time (option, default = 5 day)
+* healthcare capacity (option, default = The mortality rate is halved.)
+* choose policy (option, default = Free)
+
+  Free, Attempted quarantine, Moderate distancing, Extensive distancing
+
+* custom policy (option, default = Free)
+
+  * quarantine capacity (0 for no quarantine)
+  * accept probability of quarantine
+  * threshold of first human mobility (infected people, reduced mobility) (e.g. More than 10% of people are infected, and all people's activity is halved.)
+  * threshold of second human mobility (infected people, reduced mobility) (Enhanced first human mobility)
+  * threshold of free second human mobility (**percentage**) (if less than **percentage** poeple are infected, free the restriction of second human mobility)
+  * threshold of free first human mobility (**percentage**) (if less than **percentage** poeple are infected, free the restriction of first human mobility)
+
+Output
+^^^^^^^^^^^^
+* Animation of the virus spreading process.
+* The distribution graph of virus spread.
+* The number of uninfected people.
+* The number of recovered people.
+* The number of death people.
+
+Mathematical Model Description
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+| This is a dynamicial system. A dynamic system is a fixed rule that describes how all points in a fixed space change over time.
```

Comment:
I think you are closer to come up with some equations to describe your system.  Working them out should help you understand more about the feasibility of the design and implementation.  It makes a lot of difference to be able to code by equations.

But I would say do not spend more than a week on the equations.  Our goal is to develop the software, not a new theory.

Reply:
Okay, I will try to summarize an equation, but I won't spend a lot of time on it.